vj4/app.py

```diff
@@ -33,13 +34,43 @@
                help='Expire time for changemail token, in seconds.')
 options.define('url_prefix', default='https://vijos.org', help='URL prefix.')
 options.define('cdn_prefix', default='/', help='CDN prefix.')
+options.define('sentry_dsn', default='', help='Sentry integration DSN.')
 
 _logger = logging.getLogger(__name__)
 
 
+class SentryMiddleware(aiohttp_sentry.SentryMiddleware): # For getting a correct IP
+  async def get_extra_data(self, request):
+    return {
+      'request': {
+        'query_string': request.query_string,
+        'cookies': request.headers.get('Cookie', ''),
+        'headers': dict(request.headers),
+        'url': request.path,
+        'method': request.method,
+        'scheme': request.scheme,
+        'env': {
+          'REMOTE_ADDR': request.headers.get(options.ip_header) if options.ip_header else request.transport.get_extra_info('peername')[0]
```

Comment:
line width > 100

Reply:
We can try to use the same code for this and `Handler.remote_ip`.